lib/engine/game/g_1817_na.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require_relative 'g_1817'
+require_relative '../config/game/g_1817_na'
+
+module Engine
+  module Game
+    class G1817NA < G1817
+      load_from_json(Config::Game::G1817NA::JSON)
+
+      DEV_STAGE = :prealpha
+      GAME_PUBLISHER = nil
+      PITTSBURGH_PRIVATE_NAME = 'DTC'
+      PITTSBURGH_PRIVATE_HEX = 'F14'
+
+      GAME_LOCATION = 'North America'
+      SEED_MONEY = 150
+      GAME_RULES_URL = {
+        '1817NA' => 'https://docs.google.com/document/d/1b1qmHoyLnzBo8SRV8Ff17iDWnB7UWNbIsOyDADT0-zY/view',
+        '1817 Rules' => 'https://drive.google.com/file/d/0B1SWz2pNe2eAbnI4NVhpQXV4V0k/view',
+      }.freeze
+      GAME_DESIGNER = 'Mark Voyer'
+
+      def self.title
+        '1817NA'
+      end
+
+      # Not genericifying 1817's loan logic just so it can be kept simpler, at least for now
+      def init_loans
+        @loan_value = 100
```

Comment:
why do you need this to be an instance variable?

Reply:
The implementation of 1817 has it as an instance variable and to maximize reuse either g_1817.rb needed to be refactored or the behavior copied

For the case of loans, I chose to copy the behavior because the logic was complicated enough that I didn't want to meddle in it if I could avoid it

In the case of the Pittsburgh private, the logic was simple enough I was comfortable refactoring it to use constants which `G1817NA` overrides